src/main/recorder.ts

```diff
@@ -1,14 +1,10 @@
 import { Metadata } from './logutils';
 import { writeMetadataFile, runSizeMonitor, getNewestVideo, deleteVideo, cutVideo, addColor, getSortedVideos } from './util';
 import { mainWindow }  from './main';
-import path from 'path';
 import { AppStatus } from './types';
 
 const obsRecorder = require('./obsRecorder');
 import fs from 'fs';
-import glob from 'glob';
```

Comment:
Is this right? I thought you had added these

Reply:
`Recorder` no longer uses these - it uses `util.getSortedVideos()` instead :)

Reply:
I might have added them in a previous PR before I let `getSortedVideos()` do all the work :)